### PR TITLE
chore: Improving FS Error Message for op_realpath_sync and op_realpath_async

### PR DESCRIPTION
**Pull request by WingZer0o**
#17526

### CODE REVIEW

File: ext/fs/lib.rs [hunk 1]
```diff
@@ -1488,7 +1493,12 @@ where
     debug!("op_realpath_async {}", path.display());
     // corresponds to the realpath on Unix and
     // CreateFile and GetFinalPathNameByHandle on Windows
-    let realpath = canonicalize_path(&path)?;
+    let realpath = canonicalize_path(&path).map_err(|error| {
+      default_err_mapper(
+        error,
+        format!("op_realpath_async '{}', Deno.realPath", path.display()),
```
Comment (aapoalas) [inline ext/fs/lib.rs:1499]: **issue:** Other `default_err_mapper` calls do not include the `Deno.realPath` method name in the formatted error message. I think it wold be proper here to align with the others and remove the extra JS side method name mention.

File: ext/fs/lib.rs [hunk 2]
```diff
@@ -1456,7 +1456,12 @@ where
   debug!("op_realpath_sync {}", path.display());
   // corresponds to the realpath on Unix and
   // CreateFile and GetFinalPathNameByHandle on Windows
-  let realpath = canonicalize_path(&path)?;
+  let realpath = canonicalize_path(&path).map_err(|error| {
+    default_err_mapper(
+      error,
+      format!("op_realpath_sync '{}', Deno.realPathSync", path.display()),
```
Comment (aapoalas) [inline ext/fs/lib.rs:1462]: **issue:** Same thing here.

**Review by aapoalas — CHANGES_REQUESTED** [review]
Looks mostly good but a nitpick kind of issue regarding the formatted error messages should be resolved before merging to align with all the other FS error messages.

File: ext/fs/lib.rs (hunk 1 above)
Comment (WingZer0o) [reply ext/fs/lib.rs:1499]: I can change this up sometime today.

File: ext/fs/lib.rs (hunk 2 above)
Comment (WingZer0o) [reply ext/fs/lib.rs]: Fixed both of these up.

Comment (aapoalas) [reply ext/fs/lib.rs]: Thank you <3

**Review by aapoalas — APPROVED** [review]
Thank you very much for the fix!

**Pull request merged**